src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -353,6 +355,7 @@ void SwerveChassis::ResetPose(const Pose2d &pose)
 {
     ZeroAlignSwerveModules();
     Rotation2d rot2d{pose.Rotation().Degrees()};
+    SetStoredHeading(pose.Rotation().Degrees());
```

Comment:
Normally, I'd just use the variable you just set, but this works too (probably just a little slower)